packages/codegen.go/src/operations.ts

```diff
@@ -322,7 +322,7 @@ function generateConstructors(client: go.Client, imports: ImportManager): string
 
     // add client options last
     ctorParams.push(`options ${helpers.formatParameterTypeName(clientOptions)}`);
-    paramDocs.push(helpers.formatCommentAsBulletItem('options', clientOptions.docs));
+    paramDocs.push(helpers.formatCommentAsBulletItem('options', { summary: 'Contains optional client configuration. Pass nil to accept the default values.' }));
```

Comment:
We lose the link to the type with this change. Won't that make it harder to see what options are available? I like the new syntax but would be nice if we had a link to the *Options type.

//   - options - ### AccessClientOptions contains the optional values for creating a [AccessClient]

Reply:
The param types in the function signature will automatically link to the type.  Here's an example.

https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v7#ClientFactory